test/test.js

```diff
@@ -123,3 +123,69 @@ describe('dvb.find "Zellescher Weg"', function() {
         });
     });
 });
+
+describe('dvb.pins "5654791, 4620310, 5657216, 4623119, stop"', function() {
+    
```

Comment:
You can store the responses to test/data and mock these using mockRequest(), this speeds up and stabilizes the unit-tests.